src/test/resources/application-it.yml

```diff
@@ -39,3 +39,10 @@ header:
     x-okapi-user-id:
       exclude:
         base-paths: /tests, /admin
+folio:
+  environment: test
+  system-user:
+    enabled: false
+    username: test
```

Comment:
remove username and password please

Reply:
Breaks all integration tests with error `Could not bind properties to 'SystemUserProperties' : prefix=folio.system-user, ignoreInvalidFields=false, ignoreUnknownFields=true`